packages/zoe/src/issuerTable.js

```diff
@@ -0,0 +1,126 @@
+// @ts-check
+
+import { assert } from '@agoric/assert';
+import { E } from '@agoric/eventual-send';
+
+import makeWeakStore from '@agoric/weak-store';
+import makeAmountMath from '@agoric/ertp/src/amountMath';
+import { makeTable, makeValidateProperties } from './table';
+
+import '../exported';
+import './internal-types';
+
+/**
+ * @template K,V
+ * @typedef {import('@agoric/weak-store').WeakStore<K,V>} WeakStore
+ */
+
+// Issuer Table key: brand
+// Columns: issuer | amountMath | brand
```

Comment:
```suggestion
// Issuer Table key: brand
// Columns: brand | issuer | amountMath
```

* The columns should be listed in the same order as on the call to `makeValidateProperties`.
* The key should come first in both.

However, again, if this is the only remaining use of tables, reusable abstractions like `makeValidateProperties` for this one use seems like overkill.